doc/conf.py

```diff
@@ -345,11 +345,11 @@
 		('boards/arm/nrf52840_pca10056/doc/nrf52840_pca10056/index', 'boards/arm/nrf52840dk_nrf52840/doc/index'),
 		('boards/arm/nrf9160_pca10090/doc/nrf9160_pca10090/index', 'boards/arm/nrf9160dk_nrf9160/doc/index'),
 		('boards/arm/nrf52840_pca10090/doc/nrf52840_pca10090/index', 'boards/arm/nrf9160dk_nrf52840/doc/index'),
-		('boards/arm/nrf52840_pca10059/doc/nrf52840_pca10059', 'boards/arm/nrf52840_pca10059/doc/index'),
+		('boards/arm/nrf52840_pca10059/doc/nrf52840_pca10059', 'boards/arm/nrf52840dongle_nrf52840/doc/index'),
```

Comment:
You need to add an additional entry for each one you modify here, see https://github.com/zephyrproject-rtos/zephyr/pull/23447#discussion_r396496580.

Reply:
yes :)

Reply:
Done